test/tests/test_harness/gold/b_csvdiff_comparison_out.csv

```diff
@@ -0,0 +1,7 @@
+time,u_field
```

Comment:
I'm not sure I like your override tests. If you only have one field to override, it's equivalent to a global override. Can you perhaps add at least one more column to these tests to verify that they actually behave differently than the global versions?